src/layers/domain/core/common.py

```diff
@@ -0,0 +1,20 @@
+from typing import Iterable, TypeVar
```

Comment:
this file isn't used

Reply:
How did you discover this?  I can't see a way to find files that are unused.  What am I missing?